readthedocs/templates/base.html

```diff
@@ -85,17 +85,13 @@
 
         {% block footer-content %}
         <p>
-        Copyright {% getCopyrightYears %}.
-
-        {% blocktrans with "<a href='http://ericholscher.com/'>Eric Holscher</a>" as eric and "<a href='http://charlesleifer.com/'>Charles Leifer<a>" as charles and "<a href='http://bobbygrace.info/'>Bobby Grace</a>" as bobby and "<a href='http://djangodash.com/'>Django Dash</a>" as djangodash %}
-          Created by {{ eric }}, {{ charles }}, and {{ bobby }} for the 2010 {{ djangodash }}.
-        {% endblocktrans %}
+          Copyright {% getCopyrightYears %} Read the Docs, Inc
```

Comment:
Ugh copyright law. Not even sure if this makes sense. A couple issues:

* RTD Inc formation was a couple years into the project, this displays the start year
* AFAIK, copyright date is not necessary at all here
* We have multiple authors, not sure if this should instead read "respective owners"?
* Is a copyright even necessary?